contracts/src/WrappedWbtc.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity 0.8.24;
+
+import "openzeppelin-contracts/contracts/token/ERC20/extensions/ERC20Wrapper.sol";
+
+contract WrappedWbtc is ERC20Wrapper {
+    using SafeERC20 for IERC20;
+
+    uint256 private constant _DECIMALS_DIFF = 10;
+    address private constant _WBTC = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
+
+    constructor() ERC20Wrapper(IERC20(_WBTC)) ERC20("Wrapped WBTC", "WWBTC") {
```

Comment:
Have you tough about naming this one WBTC18?

Reply:
https://github.com/johnnyonline/USA.d/pull/2/commits/f149d23486abd6a67b10ffd8dbe5ce8daf103c30